custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -47,6 +48,9 @@ def _predict_or_transform(self, logger=None):
         file_key = "X"
         filestorage = request.files.get(file_key)
 
+        target_key = "y"
```

Comment:
Passing data in X and y, in particular, and feature in general, requires a lot of documentation.

Reply:
Also mention, that as long as two files have to be submitted, form-data params have to be used.